apimatic_faraday_client_adapter.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name = 'apimatic_faraday_client_adapter'
-  s.version = '0.1.1'
+  s.version = '0.1.2'
```

Comment:
Blocker: Kindly revert this version update change since this will be done in a separate PR when we publish the gem.